hack/ES_Upload.py

```diff
@@ -0,0 +1,72 @@
+from opensearchpy import OpenSearch
+import json
+import sys
+
+def payload_constructor(data,action):
+    action_string = json.dumps(action) + "\n"
+
+    payload_string=""
+    count=0
+    for line in data:
+        payload_string += action_string
+        this_line = json.dumps(line) + "\n"
+        payload_string += this_line
+        count+=1
+        if count == 100:
+          response=client.bulk(body=payload_string,index="ocm-api-data")
+          print(response)
+          count=0
+          payload_string=""
+        
+    return payload_string
+
+
+
+filename= sys.argv[1]
+
+
+
+# Create the client with SSL/TLS enabled, but hostname verification disabled.
+client = OpenSearch(
+    hosts = [{'host':'perf-results-elastic.apps.observability.perfscale.devcluster.openshift.com', 'port':443}],
```

Comment:
Change the host to our dev host `https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com/`